.eslintrc.json

```diff
@@ -5,6 +5,7 @@
         "commonjs": true
     },
     "rules": {
+        "no-unused-vars": ["error", {"vars": "all", "args": "none"}],
```

Comment:
I would prefer if we were explicit in when an argument is intended to be unused, for example doing `function (foo, _bar) {/* things using foo but not bar */ }`. It looks like eslint supports that via http://eslint.org/docs/2.0.0/rules/no-unused-vars#ignore-identifiers-that-match-specific-patterns